src/assets/icons/icon-health.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" version="1"><g fill="none" fill-rule="evenodd"><path d="M0 0h32v32H0z"/><path d="M3 15c-2-1-3-4-3-6 0-5 4-9 8-9 3 0 5 1 7 3 2-2 4-3 7-3 4 0 8 4 8 9 0 2-1 5-3 6L16 28h-2z" fill="#ff8e8e"/><path stroke="#F52D2D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="M7 14h3l3-6 3 14 4-8h3"/></g></svg>
+
```

Comment:
this line is not necessary ;) @ptbrowne remove it on others files https://github.com/cozy/cozy-banks/pull/907/files#diff-a0106025d772a2f6c68f3ce1cae8c0dc